### PR TITLE
exicom23and22

### PR DESCRIPTION
**Pull request by jubuhan**

### CODE REVIEW

**Comment by jubuhan** [comment]
Exicome Details 22 and 23

**Review by anima-regem — CHANGES_REQUESTED** [review]
@jubuhan Can you please increase the sizes of the images on the **Execom 23** and **Execom 24** posters? Those landscape images are not clearly visible. If possible **you can take full screen** with for those banner type images. `w-full` or `w-1/2` whichever looks good will be fine.

File: src/components/Execom23/Execom23.jsx
Comment (anima-regem) [inline src/components/Execom23/Execom23.jsx:1]: `w-full` will be better for the banner type images.

File: src/components/Exicom22/Exicom22.jsx
Comment (anima-regem) [inline src/components/Exicom22/Exicom22.jsx:1]: `w-full` will be better for the banner type images.

**Review by anima-regem — APPROVED** [review]
Looks good!

**Pull request merged**